### DISCUSSION TITLE
Looking for tutorials

### DISCUSSION THREAD

**Discussion by HemiBob** [snapcast/snapcast#764] [Q&A]
I'd love to give this a go but can't work out how to configure it, there seems to be very little advice out there for my situation.
I have Volumio running on a Pi with an IQaudiO DAC+ and this works well connected to local speakers.
Now I want to be able to stream that audio to other rooms and thought Snapcast should be able to do this.
So I added the Snapcast plugin to Volumio and installed Snapcast clients to various devices around the house but I can't get it to work and I don't really understand all the jargon involved.
I entered the IP address of the Pi in the various clients but always get "refused connections" etc
I tried "scan for servers" but this didn't find any.
I read the documentation so I understand the theory, it's just I can't find any tutorials to follow.
I went down a rabbit hole looking for /etc/config files etc but my clients are Android and Chromebooks so I soon got stuck.

**Comment by TekniskSupport** [comment]
Are you sure that the snapcast server part of the plugin is running? (if not, and you can't get it running make sure it's not a problem with the plugin itself (you may want to file the issue in the [plugin repo](https://github.com/Saiyato/volumio-snapcast-plugin) instead))

Usually the clients autodiscover the server and add itself so it shouldn't be very hard getting started.

**Reply by HemiBob** [reply]
No, I can't be sure if it's running because I've never seen a working one. 
But the plugin status page does say "SnapCast 2.4.3 - On - Active" in green.
But that doesn't mean I've set it up correctly I guess.

The "Server" page of the plugin has these settings in it:
Enable SnapServer - On
Name Of The Main Stream - SnapServer
Sample Rate - 48000Hz
Bit Depth - 16bits
Sound Channels (2 = Stereo) - 2
Codec To Use - I've tried PCM, Flac and OGC
Additional CLI Commands - empty

I have now got SSH access to the Pi and found a config file which looks like it could be for SnapCast.
This is what's in it, but it means nothing to me :-(
```
        #SNAPCAST
        pcm.!snapcast {
                type plug
                slave.pcm snapConverter
        }

        pcm.snapConverter {
                type rate
                slave {
                        pcm writeFile # Direct to the plugin which will write to a file
                        format S16_LE
                        rate 48000
                }
        }

        pcm.writeFile {
                type file
                slave.pcm null
                file "/tmp/snapfifo"
                format "raw"
        }
        #ENDOFSNAPCAST
```

**Comment by TekniskSupport** [comment]
To me that looks like an alsa config file

Now that you have access over ssh, perhaps you can troubleshoot using `ps -aux` to see running processes
and maybe trying to start the snapserver manually to view the output

But i honestly think you can get better help from the developer of the plugin

**Comment by Sirnut** [comment]
Assuming you haven't gotten this running quite yet, I'll say it's very iffy on how to get it running properly on volumio. Right now the release that is installed by the plugin is the 15.0 which is quite dated. Saiyato is the one developing the plugin, but right now he's working on a new version of the plugin based on docker so the existing volumio plugin likely wont be updated. I've found the best way to get snapcast working with volumio is to install a fresh build of volumio and then install the plugin from the repository. I have a forked repository that I've been playing around with to try and streamline the process however it's a WIP
